scss/_regions/_modal.scss

```diff
@@ -62,12 +62,16 @@
   position: absolute;
   top: 0;
   right: $base-spacing;
-  font-size: 45px;
+  font-size: 36px;
```

Comment:
They're sort of hackish since they're for an icon rather than plain text, and also to compensate for the multiplication sign "close" button being a pretty small glyph compared to other characters.